portals/api-portal/Makefile

```
@@ -78,6 +78,20 @@ build: ## Build API Portal Docker image
 		--load \
 		.
 
+# Local dev registry knobs — mirror the Dakshina pattern used across
+# apim-saas/services/apip-* (services/apip-platform-api/Makefile).
+LOCAL_REGISTRY ?= localhost:10082
+LOCAL_TAG      ?= dev
+
+.PHONY: build-local
+build-local: ## Build and push to the local k3d registry as :dev (for wso2cloud local dev)
```

checkmake (0.3.2)

[warning] Target body for "build-local" exceeds allowed length of 5 lines (6).

(maxbodylength)